src/consensus/pbft/pbft.h

```diff
@@ -119,9 +119,22 @@ namespace pbft
         nodes[to] = end_idx;
       }
 
-      // The host will append log entries to this message when it is
-      // sent to the destination node.
-      n2n_channels->send_authenticated(ccf::NodeMsgType::consensus_msg, to, ae);
+      auto tmsg = std::make_unique<enclave::Tmsg<SendAuthenticatedAEMsg>>(
```

Comment:
What's the performance gain associated with this PR? It is not clear from the cimetrics report.

Reply:
This is because in the pipeline jobs we are running with 1 thread. This is because of the OE bug :(

Reply:
So that got merged: https://github.com/openenclave/openenclave/pull/2656

I don't think we ought to wait for the next release on this either, let's refresh both the docker CI and the perf kit.